channel-jira-server/src/main/java/com/synopsys/integration/alert/channel/jira/server/validator/JiraServerGlobalConfigurationValidator.java

```diff
@@ -50,7 +51,7 @@ public ValidationResponseModel validate(JiraServerGlobalConfigModel model, Strin
                 statuses.add(AlertFieldStatus.error("url", e.getMessage()));
             }
         }
-        if (StringUtils.isBlank(model.getUserName())) {
+        if (model.getUserName().isEmpty() || StringUtils.isBlank(model.getUserName().get())) {
```

Comment:
I think here we should check if model.getAuthorizationMethod() is BASIC first so it doesnt fail if it's set to PAT.